### DISCUSSION TITLE
Swap button values

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#128] [General]
Next version will be able to do this:

https://github.com/user-attachments/assets/0c2eaba6-7859-483e-8f63-e7922f884dfe

Values are checked to make sure the buttons are compatible.

**Comment by N3C2L** [comment]
Great enhancement, thanks! And this new shortcuts for GH even work when executed from internal urls like "about:" or "moz-url", for example switching tabs. But maybe some users might be confused where they have to assign these browser commands to shortcut keys. You could add an info in the headline on top of this "browser" setting where you point them to the about:addons page and the "manage extension shortcuts" button on the top right dropdown.

**Reply by madprops** [reply]
Ok good idea
